File: Find_duplicates_in_an_array/code.cpp
class Solution{
  public:
    vector<int> duplicates(int arr[], int n) {
        // code here
    vector<int> res;
        sort(arr, arr+n);
        int i=0;
        int x = 1;
        while(i<n){
            if(arr[i] == arr[i+1]){
                x= 1;
                res.push_back(arr[i]);
                while(arr[i] == arr[i+x]){
                    x++;
                }
                i+=x;
            }
            else{
                i++;
            }
        }
        if(res.size() == 0){
            res.push_back(-1);
        }
        return res;
    }
};

File: Count_the_digits_that_divide_a_number/code.java
class Solution{
    static int evenlyDivides(int N){
        // code here
        int copy=N;
        int count=0;
        while(copy>0)
        {
            int d=copy%10;
            copy=copy/10;
            if(d==0)
            continue;
            if(N%d==0)
            count++;
        }
        return count;
    }
}
